pnpm-lock.yaml

```diff
@@ -6199,10 +6242,18 @@ packages:
     resolution: {integrity: sha512-GuUdqkyyzQI5RMIWkHhvTWLCyLo1jNK3vzkSyaExH5kHPDHcuL2VOpHjmMY+y3+NC69qAKToBqldTBgYeLSr9Q==}
     engines: {node: ^18.18.0 || ^20.9.0 || >=21.1.0}
 
+  '@eslint/eslintrc@2.1.4':
+    resolution: {integrity: sha512-269Z39MS6wVJtsoUl10L60WdkhJVdPG24Q4eZTH3nnF6lpvSShEK3wQjDX9JRWAUPvPh7COouPpU9IrqaZFvtQ==}
+    engines: {node: ^12.22.0 || ^14.17.0 || >=16.0.0}
+
   '@eslint/eslintrc@3.2.0':
     resolution: {integrity: sha512-grOjVNN8P3hjJn/eIETF1wwd12DdnwFDoyceUJLYYdkpbwq3nLi+4fqrTAONx7XDALqlL220wC/RHSC/QTI/0w==}
     engines: {node: ^18.18.0 || ^20.9.0 || >=21.1.0}
 
+  '@eslint/js@8.57.1':
```

Comment:
It would be really nice if we could avoid bringing in outdated deps like this.

We've long avoided `@wordpress/scripts` for that reason. It wants to bring its own versions of a lot of things which we already have set up ourselves, and we don't want to be stuck matching their versions for everything to avoid unnecessarily duplicated deps.

Reply:
Yeah, I did notice that. I installed it only for that playwright config. Should we copy that config file instead?

Reply:
I have removed it as a dependency and copied the best bit of that playwright config over.